include/Hardmon.hpp

```diff
@@ -0,0 +1,230 @@
+#ifndef VCU_HARDMON_HPP
+#define VCU_HARDMON_HPP
+
+#include <EVT/io/CAN.hpp>
+#include <EVT/io/CANDevice.hpp>
+#include <EVT/io/CANOpenMacros.hpp>
+#include <EVT/io/GPIO.hpp>
+#include <EVT/io/pin.hpp>
+#include <EVT/io/types/CANMessage.hpp>
+#include <dev/PowertrainCAN.hpp>
+#include <models/Hardmon_Model.hpp>
+
+namespace IO = EVT::core::IO;
+
+namespace VCU {
+
+/**
+ * Driver for the Hardware Monitor
+ */
+
+class Hardmon : public CANDevice {
+public:
+    /**
+     * Struct that contains all the GPIOs that an instance of this class requires.
+     */
+    struct reqGPIO {
+        //model input pins
+        IO::GPIO& ignitionCheckGPIO;
+        IO::GPIO& ignition3V3GPIO;
+        IO::GPIO& lvssStatus3V3GPIO;
+        IO::GPIO& mcStatusGPIO;
+
+        IO::GPIO& ucStateZeroGPIO;
+        IO::GPIO& ucStateOneGPIO;
+        IO::GPIO& ucStateTwoGPIO;
+        IO::GPIO& ucStateThreeGPIO;
+
+        IO::GPIO& eStopCheckGPIO;
+        IO::GPIO& watchdogGPIO;
+        IO::GPIO& eStop3V3GPIO;
+
+        //model outputs pins
+        IO::GPIO& lvssEnableOverrideGPIO;
+        IO::GPIO& mcToggleNegativeGPIO;
+        IO::GPIO& mcTogglePositiveGPIO;
+        IO::GPIO& mcToggleOverrideGPIO;
+        IO::GPIO& ucResetGPIO;
+        IO::GPIO& lvssEnableHardmonGPIO;
+        IO::GPIO& hmFaultGPIO;
+    };
+
+    /**
```

Comment:
The double asterisk indicates this is a comment you want Doxygen to pick up. Here, the comment is meant to help understand the code but likely not for Doxygen
```suggestion
    /*
```